### ISSUE TITLE
besca.datasets.pbmc_storage_processed() is broken

### ISSUE THREAD

**Issue by Accio** [bedapub/besca#27]
It is used in the standard workflow, but the file (besca/datasets/data/pbmc_storage_processed_downsampled.h5ad) cannot be found.

This prevents the standard workflow being executed without error.

**Comment by ajulienla** [comment]
To add to this, one decision is reached, all datasets should be reloaded accordingly. 
Currently 
```
 filename = pkg_resources.resource_filename('besca', 'datasets/data/pbmc3k_raw.h5ad', backup_url='http
s://zenodo.org/record/3752813/files/pbmc3k_raw.h5ad?download=1')
```
does not work. backup_url is not an accepted  parameter.

This has to be fix one Zenodo permisions are settled.

**Comment by hatjek** [comment]
It should work now. @ajulienla could you please try again?

**Comment by ajulienla** [comment]
Yes the URL works. The command does not.

**Comment by hatjek** [comment]
I fixed this in #39. Still, depending on the python installation and the firewall setup one can run into a [SSL: CERTIFICATE_VERIFY_FAILED] error.
